health/healthcheck.go

```diff
@@ -87,6 +95,18 @@ func (h *HealthCheck) writeQueueCheck() fthealth.Check {
 	}
 }
 
+func (h *HealthCheck) kafkaConsumerMonitoringCheck() fthealth.Check {
+	return fthealth.Check{
+		ID:               "kafka-consumer-lag-check",
+		Name:             "Check Kafka consumer status",
+		Severity:         3,
+		BusinessImpact:   "Consumer is lagging behind when reading messages. Ingestion is delayed.",
+		TechnicalSummary: "Messages awaiting handling exceed the configured lag tolerance. Check if Kafka consumer is stuck.",
```

Comment:
**nitpick(non-blocking):** The [constant provided by kafka-client-go](https://github.com/Financial-Times/kafka-client-go/blob/v3/consumer.go#L26) can be used here